content/en/docs/contribute/participating.md

```diff
@@ -108,39 +104,45 @@ in the Kubernetes organization. Follow these steps:
     GitHub issue to show approval and then closes the GitHub issue.
     Congratulations, you are now a member!
 
-If for some reason your membership request is not accepted right away, the
+If for some reason your membership request is not accepted, the
 membership committee provides information or steps to take before applying
 again.
 
-### Reviewers
+## Reviewers
 
 Reviewers are members of the
 [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
-GitHub group. See [Teams and groups within SIG Docs](#teams-and-groups-within-sig-docs).
-
-Reviewers review documentation pull requests and provide feedback on proposed
-changes.
-
-Automation assigns reviewers to pull requests, and contributors can request a
+GitHub group. Reviewers review documentation pull requests and provide feedback on proposed
+changes. Reviewers can do the following:
+
+- Everything listed under [Anyone](#anyone) and [Members](#members)
+- Document new features
+- Triage and categorize issues
+- Review pull requests and provide binding feedback
+- Create diagrams, graphics assets, and embeddable screencasts / videos
+- Localization
+- Contribute to other Kubernetes repositories as a docs representative
```

Comment:
We don't really have representatives as such. Anyone can advocate for better docs.
```suggestion
```